README.md

```diff
@@ -399,64 +140,97 @@ render() {
 }
 ```
 
-Navigation dots can also be further customized with your own styles.
-```javascript
-import Carousel, { Dots } from '@brainhubeu/react-carousel';
-import '@brainhubeu/react-carousel/lib/style.css';
-
-// ...
-
-constructor(props) {
-  super(props);
-  this.state = {
-    value: 0,
-  };
-}
-
-onChange = value => this.setState({ value });
-
-render() {
-  return (
-    <div>
-      <Carousel
-        value={this.state.value}
-        onChange={this.onChange}
-      >
-        <img className="img-example" src={someImage} />
-        ...
-        <img className="img-example" src={anotherImage} />
-      </Carousel>
-      <Dots value={this.state.value} onChange={this.onChange} number={12} className={yourOwnClassName} />
-    </div>
-  );
-}
-```
-
-## Unit tests
+[![gif](readme/assets/thumbnails.gif)](https://github.com/brainhubeu/react-carousel/)
+
+## Props
+You can access a clickable demo with many examples and a [live code editor](https://brainhubeu.github.io/react-carousel/) by clicking on a Prop name.
+
+### Carousel props
+
+| Prop | Type | Description |
+| --- | --- | --- |
+| [**addArrowClickHandler**](https://brainhubeu.github.io/react-carousel/docs/examples/customArrows) | *Boolean* | Has to be added for arrowLeft and arrowRight to work |
+| [**animationSpeed**](https://brainhubeu.github.io/react-carousel/docs/examples/animation) | *Number* | Determines transition duration in milliseconds |
+| [**arrowLeft**](https://brainhubeu.github.io/react-carousel/docs/examples/customArrows) | *React element* | To be used instead of default left arrow (if you provide these custom arrows, you don't have to use arrows prop) |
+| [**arrowRight**](https://brainhubeu.github.io/react-carousel/docs/examples/customArrows) | *React element* | To be used instead of default right arrow (if you provide these custom arrows, you don't have to use arrows prop) |
+| [**arrows**](https://brainhubeu.github.io/react-carousel/docs/examples/defaultArrows) | *Boolean* | Renders default arrows |
```

Comment:
For each Boolean prop, could you formulate the description to start with the following?
```
If set to `true`, it...
```
and could you add `Otherwise, ...` when it's not obvious?

Reply:
I'm not sure about it. I saw many good repositories without this additional phrase and I think we don't need to extend our descriptions.

Reply:
or even two separate columns for booleans:
- `behavior when true`
- `behavior when false`

Reply:
I'm sorry, this not a common solution. I think that will only add more text to read and this is redundant. If you do not agree, we can vote on Thursday's meeting.

Reply:
I can accept the current solution for now and we could discuss this on Thursday.